src/sections/Home/Partners-home/partners-home-data.js

```diff
@@ -59,10 +59,16 @@ export const partners = [
     imageRoute: "/partners#uem",
     innerDivStyle: "partner__block__inner"
   }, {
-    name:"VMware",
+    name: "VMware",
     imageLink: vmware,
-    imageRoute:"",
-    innerDivStyle:"partner__block__inner"
-    
+    imageRoute: "",
+    innerDivStyle: "partner__block__inner"
+
+  }, {
+    name: "VMware",
+    imageLink: vmware,
+    imageRoute: "/partners#VMware",
+    innerDivStyle: "partner__block__inner",
```

Comment:
We already have VMware added just above this, no need to add it again.
Just update the `imageRoute` above to point to `/partners#VMware`